doc/fbchkdoc/delextra.bas

```diff
@@ -54,23 +71,67 @@ function ReadIndex( byref f as string ) as integer
 	return FALSE
 end function
 
+''
+function ScanForDeleteMe _
+	( _
+		filename as const string _
+	) as boolean
+
+	dim sBody as string
+	dim chk as string = "!!! DELETE ME !!!"
+
+	sBody = LoadFileAsString( filename )
+	if( left( ucase( ltrim( sBody, any " " & chr(9))), len( chk ) ) = chk ) then
+		function = true
+	else
+		function = false
+	end if
+
+end function
+
 ''
 sub DeleteExtraFiles _
 	( _
-		byref path as string, _
-		byval isgit as integer, _
-		byval nodelete as integer _
+		byref path as const string, _
+		byval isgit as boolean, _
+		byval nodelete as boolean, _
+		byval doscan as boolean, _
+		byval bHTML as boolean, _
+		byref wiki_url as const string _
 	)
 
 	dim d as string, i as integer
+	dim remove as boolean
+	dim pagename as string
+
+	if( bHTML ) then
+		loghtml( "", TRUE )
+		loghtml( "<html><body>" )
+	end if
+
 	d = dir( path + "*.wakka" )
 	while( d > "" )
+		
+		pagename = left( d, len(d) - len(".wakka") )
+
 		if( filehash.test( lcase(d) ) = FALSE ) then
```

Comment:
maybe this `if` can be turned into a function, then e.g. `var remove = shouldRemove(d)`